microsoft/testsuites/network/synthetic.py

```diff
@@ -39,7 +39,7 @@ class Synthetic(TestSuite):
         """,
         priority=2,
         requirement=simple_requirement(
-            min_nic_count=8,
+            nic_count=8,
```

Comment:
why not use min_nic_count?

Reply:
hmm, seems I misunderstand the issue #1907, will push another fix